### PR TITLE
Feat: upgrade BLT teleport contracts

### PR DESCRIPTION
**Pull request by andrew54068**
same as title

asana:
https://app.asana.com/0/1176207063966309/1208032209432443/f

check list:
- [x] TeleportCustodyBSC
- [x] TeleportCustodyEthereum
- [x] TeleportCustodySolana
- [x] TeleportCustoryAptos

### CODE REVIEW

**Review by joshuahannan — APPROVED** [review]

File: contracts/flow/teleport/TeleportCustody.cdc [hunk 1]
```diff
@@ -1,176 +1,224 @@
...
+		return TeleportCustody.lockVault.balance
+	}
+	
+	init() { 
+		self.teleportAddressLength = 20
+		self.teleportTxHashLength = 64
+		self.AdminStoragePath = /storage/TeleportCustodyAdmin
+		self.TeleportAdminStoragePath = /storage/TeleportCustodyTeleportAdmin
+		self.TeleportAdminTeleportUserPath = /public/TeleportCustodyTeleportUser
+		self.TeleportAdminTeleportControlPath = /private/TeleportCustodyTeleportControl
```
Comment (imjacklai) [inline contracts/flow/teleport/TeleportCustody.cdc:217]: The private path is removed in Cadence 1.0. Please check the other files as well.

**Review by imjacklai — CHANGES_REQUESTED** [review]

File: contracts/flow/teleport/TeleportCustody.cdc
Comment (kbehouse) [inline contracts/flow/teleport/TeleportCustody.cdc:1]: preserve indent is better and more easily to compare old contract

File: contracts/flow/teleport/TeleportCustody.cdc (hunk 1 above)
Comment (andrew54068) [reply contracts/flow/teleport/TeleportCustody.cdc]: but this is the init function, can we modify that and should we?

**Review by imjacklai — APPROVED** [review]

File: contracts/flow/teleport/TeleportCustodyEthereum.cdc [hunk 2]
```diff
@@ -1,176 +1,224 @@
...
+    
+    access(AdminEntitlement)
+    fun updateUnlockFee(fee: UFix64) { 
       self.unlockFee = fee
     }
-
-    pub fun getFeeAmount(): UFix64 {
+    
+    access(AdminEntitlement)
+    fun getFeeAmount(): UFix64{
```
Comment (kbehouse) [inline contracts/flow/teleport/TeleportCustodyEthereum.cdc:188]: nit: Will it effect the backend to get the amount? 
(more restrict is better, so this is a nit)

**Review by kbehouse — APPROVED** [review]

File: contracts/flow/teleport/TeleportCustodyEthereum.cdc (hunk 2 above)
Comment (andrew54068) [reply contracts/flow/teleport/TeleportCustodyEthereum.cdc:188]: fixed.

**Review by scottphc — APPROVED** [review]

**Pull request merged**